### PR TITLE
[not EXTREME not DRAFT] THE LORE DROP PR

### PR DESCRIPTION
**Pull request by tiramisuapimancer**
## About The Pull Request
Makes changes to the game in order to make it comply with the lore. 
PR is draft as hell and will be worked on for a bit

## CHECKLIST OF SHIT 2 DO
[checklist itself still WIP]
- [x] Species blurbs
- [x] Rename lizards to Sarathi
- [x] rename Teshari to Kepori
- [x] rename ethereals to Elzuosa
- [x] remove Voltaic, give Elzuosa Draconic
- [x] language blurbs
(the stuff below is stuff o riginally planned for this PR that probably should be in a further PR for various reasons and so i'm not counting it anymore)
- [ ] drip
- [ ] change Common to be assembled from most systems commons
- [ ] Solarian language

## Why It's Good For The Game
lorema Balls

## Changelog
:cl:
tweak: THE LORE IS IN. species and language blurbs have been edited to reflect
tweak: Lizards renamed to Sarathi, Teshari renamed to Kepori, ethereals renamed to Elzuosa
del: Voltaic has been removed, Elzuosa speak Kalixcian (renamed draconic) now
add: kepori namegen
/:cl:

### CODE REVIEW

**Comment by Nobelium-XVIII** [comment]
noooo you cant just make a balls joke in a pr theyre meant to be serious youre gonna get repobanned!!!!

**Comment by AsciiSquid** [comment]
How do you pronounce Elzuosa

**Comment by tiramisuapimancer** [comment]
the most natural and non-fucked-up-way you can conceptualize

**Comment by Anderson-Cooper** [comment]
add 2021 lore :trollface:

**Comment by Nobelium-XVIII** [comment]
Sir are you ok

**Comment by triplezeta** [comment]
:trollface:

**Comment by Bokkiewokkie** [comment]
> Sir are you ok

They posted raw encrypted lore documents

**Comment by Nobelium-XVIII** [comment]
> > Sir are you ok
> 
> They posted raw encrypted lore documents

You fool, its now saved in my gmail
I will now begin decryption

**Comment by Nobelium-XVIII** [comment]
what does lorema mean

**Comment by ghost** [comment]
> what does lorema mean

It is an ancient incantation; if you say it 3 times marg will appear in front of you.

**Comment by Anderson-Cooper** [comment]
i tried sending trollface ascii but it failed and now people are theorizing.

**Comment by ZephyrTFA** [comment]
> i tried sending trollface ascii but it failed and now people are theorizing.

I need you to not egregiously spam on the github. This will be your only warning.

**Comment by ZephyrTFA** [comment]
First of all, you're ugly

**Comment by tiramisuapimancer** [comment]
With the fact that some of the language stuff and the drip is going to be in another PR, this is probably ready for testing (and shipping) whenever we're next up? Give me a bit to make the changelog suck less, as well

**Comment by FlamingLily** [comment]
Hell yeah love lore, gotta be one of my favourite ipcs

**Comment by tiramisuapimancer** [comment]
ok i resolved those merge conflicts through a webedit because my byond is too high version to compile shitpest rn so HPEFULLY this is fine

File: code/game/machinery/scan_gate.dm [hunk 1]
```diff
@@ -21,7 +21,7 @@
 #define SCANGATE_IPC "ipc"
 #define SCANGATE_SQUID "squid"
 #define SCANGATE_ETHEREAL "ethereal"
-#define SCANGATE_TESHARI "tesh"
+#define SCANGATE_KEPORI "tesh"
```
Comment (MarkSuckerberg) [inline code/game/machinery/scan_gate.dm]: god why didn't I fix this hnnnghgjghgjhnfgsjkdf

**Comment by MarkSuckerberg** [comment]
yolo

**Pull request merged**